### DISCUSSION TITLE
What is best way to have a "persistant" one click SD using fast-stable-diffusion?

### DISCUSSION THREAD

**Discussion by juan9999** [TheLastBen/fast-stable-diffusion#1555] [General]
When I run automatic1111 I'd like to have access to all my custom .CKPT files, VAEs, etc.

I've noticed that if I save these in the right directory structure of my google drive these are available.

Before I upgrade my google drive from the free tier so that I have enough room to save everything, can someone confirm that this is the case?

If my understanding is correct, is there a way to make running the colab notebook "one click" i.e. if the models etc are already deployed, why can't I skip all the steps other than connect to google drive and "Start Stable-Diffusion"?

**Comment by TheLastBen** [comment]
there are 4 steps, connect the drive (25s), install/update the repo (15s or less), install the requirements (20s), Load the model (0s if already exists), the start the webui, so that's 1 minute from start to finish
